src/components/panel-body/UEpage/communication/form/CommunicationRequestValidation.jsx

```diff
@@ -100,10 +110,10 @@ const ErrorDisplayer = ({ errors, prefix }) => {
   const classes = useStyles();
   return Object.entries(errors)
     .filter(([, val]) => val)
-    .map(([key, value]) => (
+    .map(([key]) => (
       <Typography
         key={`${prefix}-error-${key}`}
         className={classes.error}
-      >{`${prefix}-${key} : ${value}`}</Typography>
+      >{`${prefix} : ${key}`}</Typography>
     ));
 };
```

Comment:
A bit complicated, you filter val to avoid null, undefined or false ? and then render typo not based on val ?